bin/generate-docs.sh

```diff
@@ -74,6 +74,7 @@ function generate_scanner_docs() {
       "docs/README.DockerHub-Parser.md" \
       "${scanner_dir}/.helm-docs.gotmpl" \
       "${HELM_DOCS_DIR}/README.DockerHub-Parser.md.gotmpl"
+    # XXX: #754 Why is here no generation for ArtifactHub?
```

Comment:
Parsers don't have their own helm charts by them self but are included in the chart related to the scanner